tower-web/src/styles.scss

```diff
@@ -201,13 +198,6 @@ code {
     margin-left: 1em;
     table-layout: fixed;
 
-    @media (min-width: 1400px) {
-      max-width: 1000px;
-    }
-    @media (min-width: 1650px) {
-      max-width: 1140px;
-    }
-
     th:nth-child(1) {
```

Comment:
I remember we needed this for #97. Let me check if it's still a problem.

Reply:
Yes, now that the screen width is properly responsive and fluid after my first PR, I don't think these kinds of page overflow / width issues should be a problem any more.. 👍